src/app/shared/components/combobox/combobox.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="combo-container">
+  <div
+    (click)="onClickEventAction($event)"
+    [ngClass]="{ 'combo-open': showDropDown }"
+    class="combo-content-container">
+    <div class="combo-cards-container">
+      <ng-container *ngIf="comboValues.length > 0">
+        <div
```

Comment:
These are the clickable boxes that get added when you select a value and removed when you click the x button